### PR TITLE
Fixes #22991: Rewrite the 'technique-version' app in Elm

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://issues.rudder.io/issues/22991

This is part of the plan to completely remove AngularJs from Rudder, for reasons of security and code maintenance.

No new features have been added, this is a rewrite of the AngularJs "techniqueVersion" application in Elm.
The js files related to the old application have been removed

### CODE REVIEW

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by RaphaelGauthier** [comment]
Commit modified

File: webapp/sources/rudder/rudder-web/src/main/elm/sources/TechniqueVersion/DataTypes.elm [hunk 1]
```diff
@@ -0,0 +1,36 @@
+module TechniqueVersion.DataTypes exposing (..)
+
+import Http exposing (Error)
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-web/src/main/elm/sources/TechniqueVersion/DataTypes.elm:3]: Why there imports ? I don't think you need any

**Comment by VinceMacBuche** [comment]
Can you clean imports as much as possible ?

**Comment by RaphaelGauthier** [comment]
PR updated with a new commit

**Review by VinceMacBuche — APPROVED** [review]

**Comment by RaphaelGauthier** [comment]
OK, squash merging this PR

**Comment by RaphaelGauthier** [comment]
OK, squash merging this PR

**Review by VinceMacBuche — APPROVED** [review]

**Pull request merged**